File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 4/PRAK604-2210817210014-MuhammadQoharyRawidafany.py
kata1 = input()
kata2 = input()
b = 0;c = 0
if len(kata1) == len(kata2):
    for a in range(len(kata1)):
        if kata1[a] == kata2[a]:
            if kata1[a] == ' ':
                print(end=' ')
            else :
                print("*", end='')
                b = b+1
        else:
            print("#", end='') 
            c = c+1
    print("\n* = %d"%(b))
    print("# = %d"%(c))
    if b >= c:
        print("Pesan Asli")
    else :
        print("Pesan Palsu")
else :
    print("Panjang kalimat berbeda, pesan palsu")

File: LAPORAN PRATIKUM/MODUL7/CODEBLOKS DAN PYTHON/SOAL 4/PRAK604-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
#include <string.h>
int main()
{
    char kata1[123];
    char kata2[123];
    scanf("%[^\n]%*c", &kata1);
    scanf("%[^\n]%*c",&kata2);
    int kode,pesan,a,b=0,c=0;
    kode=strlen(kata1);
    pesan=strlen(kata2);
    if(kode ==pesan){
            for(a=0; a<kode; a++){
                if(kata1[a]==kata2[a]){
                    if(kata1[a]== ' '){
                        printf(" ");}
                    else{
                    printf("*");
                    b++;}
                }
                else{
                    printf("#");
                    c++;}
            }
            printf("\n*= %d",b);
            printf("\n#= %d",c);
             if(b>=c){
                printf("\nPesan Asli");}
                else{
                    printf("\nPesan Palsu");}
                    }
    else {
        printf("Panjang kalimat berbeda, pesan palsu");}
    return 0;
}
